### DISCUSSION TITLE
Add GSA webhook as trigger for task.

### DISCUSSION THREAD

**Discussion by birdman3131** [gameserverapp/Platform#212] [Suggestions]
![image](https://user-images.githubusercontent.com/749937/175492293-aaa988d9-9d35-41a4-954e-f6352695ebe0.png)
Add a trigger that uses the existing webhook api. https://help.gameserverapp.com/article/143-webhook-api so that tasks could be triggered externally.

**Comment by mvaessen** [comment]
As this will be abused most likely, i'm going to reject
